### ISSUE TITLE
Access denied to removable media (download and upload files) on Linux

### ISSUE THREAD

**Issue by PJRuas** [getferdi/ferdi#1975]
Problem description:
When using the Snapcraft release of Ferdi, it is not possible to use download/upload files to mounted units or removable devices within services as it denies access.

Steps to reproduce:

Install Ferdi using snap install ferdi
Login to Ferdi
Attempt to send/save a file located in a /mnt unit.
Additionally, attempting to manually connect using snap connect ferdi:removable-media fails with the error message:

error: snap "ferdi" has no plug named "removable-media"

Expected behavior
Snap connection success

Environment:

Operating System: Ubuntu 21.04

Ferdi Version: 5.6.2 (latest/stable) snapcraft release

Server: Ferdi

**Comment by mhatvan** [comment]
We have to extend the default plugs to include removable devices. On vacation now but I can provide a PR in a week.

**Comment by vraravam** [comment]
@mhatvan - when you work on this, please ensure that we can somehow test the `electron-builder` support for plugs in snap (https://github.com/getferdi/ferdi/pull/1892 was reverted due to an upstream bug)

**Comment by pohutukawa** [comment]
Yes, this is super annoying. It additionally doesn't allow to access anything under `/tmp`, which is often used for ephemeral file storage. I'd really appreciate if the snap builds would include those file system access provisions.

This has been tested on Ubuntu impish (21.10)
